src/thunderbots/software/ai/hl/stp/tactic/move.cpp

```diff
@@ -0,0 +1,15 @@
+#include "ai/hl/stp/tactic/move.h"
+
+MoveTactic::MoveTactic(const Point &destination) : destination(destination)
+{
+}
+
+std::pair<Robot, Team> MoveTactic::selectRobot(const Team &available_robots)
+{
+    return std::make_pair(available_robots.getAllRobots().at(0), available_robots);
```

Comment:
If I understood this correctly, this always selects the first robot in the vector? If this is for testing/just because it is early, you might want to add a comment explaining that.

Reply:
Just following up on what Henry said, this should have a Javadoc comment and should be called something like `selectFirstRobot` (if that's the intent anyhow).

Reply:
The return statement is just a placeholder (that goes for pretty much every return statement in the PR). Let's come back to this after I add the comments.